src/my_crypto.erl

```diff
@@ -2,12 +2,16 @@
 
 -export([hash/1, sign/2, verify/3, read_file_key/2]).
 
+-spec hash(iodata()) -> binary().
 hash(Bin) -> crypto:hash(sha256, Bin).
 
-sign(_Hash, _PrivateKey) -> sign. %%public_key:sign(Hash, none, PrivateKey).
+-spec sign(binary(), public_key:private_key()) -> binary().
+sign(_Hash, _PrivateKey) -> <<>>. %%public_key:sign(Hash, none, PrivateKey).
+
+-spec verify(binary(), binary(), public_key:public_key()) -> boolean().
 verify(_Hash, _Signature, _PubKey) -> true. %%public_key:verify(Hash, none, Signature, PubKey).
-    
 
+-spec read_file_key(private, string()) -> _.
```

Comment:
return type = :terror:

Reply:
lol it's scream not terror

got it: 😱

Reply:
🙀

Reply:
too bad there aren't stickers

![image](https://appstickers-cdn.appadvice.com/1151395835/818792242/338f5914a1a7fc3beef9b4a2a7644c97-7.gif)

Reply:
microsoft bought github instead of tencent RIP.